src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Embedded.cs

```diff
@@ -255,8 +331,12 @@ public void M(in int p)
 }";
 
             CreateCompilation(code, references: new[] { reference }).VerifyEmitDiagnostics(
-                // error CS8006: Forwarded type 'EmbeddedAttribute' conflicts with type declared in primary module of this assembly.
-                Diagnostic(ErrorCode.ERR_ForwardedTypeConflictsWithDeclaration).WithArguments("Microsoft.CodeAnalysis.EmbeddedAttribute").WithLocation(1, 1));
+                // (2,99): warning CS0436: The type 'EmbeddedAttribute' in 'injected type' conflicts with the imported type 'EmbeddedAttribute' in 'reference, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'. Using the type defined in 'injected type'.
```

Comment:
>The type 'EmbeddedAttribute' in 'injected type' [](start = 43, length = 47)

The message sounds strange: "The type 'EmbeddedAttribute' in 'injected type'"

Reply:
I agree, there are a few odd diagnostics. But they are corner case and didn't seem to warrant dedicated messages.
I didn't want to say "injected source", since technically we're not injecting source.
So I prefer to leave this as-is.